frontend/packages/dev-console/src/components/catalog/utils/catalog-utils.tsx

```diff
@@ -51,6 +51,20 @@ export const updateURLParams = (paramName: string, value: string | string[]) =>
   setURLParams(params);
 };
 
+export const getURLWithParams = (paramName: string, value: string | string[]): string => {
+  const params = new URLSearchParams(window.location.search);
+  const url = new URL(window.location.href);
+
+  if (value) {
```

Comment:
In this case if it is ever possible to get an empty array then it is better to add a check for that.
```suggestion
  if (!_.isEmpty(value)) {
```

Reply:
Empty array is not going to be a problem as we are doing `JSON.stringify` after an `isArray` check.